docker/scripts/deploy_parameters_docker.json

```diff
@@ -3,18 +3,19 @@
     "trustedSequencerURL": "http://zkevm-json-rpc:8123",
     "networkName": "zkevm",
     "version":"0.0.1",
-    "trustedSequencer":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+    "trustedSequencer": <deployerAddress>,
     "chainID": <chainID>,
+    "deployerPvtKey": <deployerPvtKey>,
     "trustedAggregator":"0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
     "trustedAggregatorTimeout": 604799,
     "pendingStateTimeout": 604799,
     "forkID": 1,
-    "admin":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
-    "zkEVMOwner": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
-    "timelockAddress": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+    "admin": <deployerAddress>,
+    "zkEVMOwner": <deployerAddress>,
+    "timelockAddress": <deployerAddress>,
     "minDelayTimelock": 10,
     "salt": "0x0000000000000000000000000000000000000000000000000000000000000000",
-    "zkEVMDeployerAddress":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
-    "initialZkEVMDeployerOwner" :"0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
-    "maticTokenAddress":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
+    "zkEVMDeployerAddress": <deployerAddress>,
+    "initialZkEVMDeployerOwner" : <deployerAddress>,
+    "maticTokenAddress": <deployerAddress>
```

Comment:
iiuc, `maticTokenAddress` should be the deployed Matic token contract address instead of an EOA. we should probably check the code to understand this field a bit more

Reply:
yes, it is the deployed Matic token contract address, and for Testnet/mock L1, it can be left blank.